src/i18n/locales/en_US/user.json

```diff
@@ -14,7 +14,8 @@
       "invalid_password": "Password incorrect!",
       "last_name_too_long": "Please limit Last Name to 50 characters or fewer.",
       "please_enter_valid_email": "Please enter a valid email address.",
-      "unknown_error": "An unknown error occurred"
+      "unknown_error": "An unknown error occurred",
```

Comment:
Please run `yarn i18next` which should parse the translation strings that are used and make sure they're added to all of the language and definition files.